### PR TITLE
DM-41219: Fix errors in dp02 and dp02 schemas

### PR DESCRIPTION
**Pull request by JeremyMcCormick**

### CODE REVIEW

File: yml/dp02_dc2.yaml [hunk 1]
```diff
@@ -8092,37 +8092,37 @@ tables:
     datatype: double
     mysql:datatype: DOUBLE
     description: Elliptical Gaussian adaptive moments
-    fits:tunit: asec^2
+    fits:tunit: arcsec2
```
Comment (gpdf) [inline yml/dp02_dc2.yaml:8095]: VOUnit requires `arcsec**2` here.

**Review by gpdf — CHANGES_REQUESTED** [review]
Please correct the units ( `arcsec2` -> `arcsec**2` ) as noted in the detailed comments in one place (but it appears multiple times).  This is what the VOUnit standard requires.

We need to see if we can figure out how to make the automated checks more strict.

File: yml/dp02_dc2.yaml (hunk 1 above)
Comment (JeremyMcCormick) [reply yml/dp02_dc2.yaml]: This should be fixed now.

File: yml/dp02_dc2.yaml [hunk 2]
```diff
@@ -8092,37 +8092,37 @@ tables:
...
     description: Elliptical Gaussian adaptive moments
-    fits:tunit: asec^2
+    fits:tunit: arcsec**2
   - name: ixyPSF
     "@id": "#DiaSource.ixyPSF"
     datatype: double
     mysql:datatype: DOUBLE
     description: Adaptive moments of the PSF model at the object position
-    fits:tunit: asec^2
+    fits:tunit: arcsec2
```
Comment (gpdf) [inline yml/dp02_dc2.yaml:8113]: Missed one here

**Review by gpdf — APPROVED** [review]
Looks good now, thank you!

**Pull request merged**